### ISSUE TITLE
Staff pick not properly translated

### ISSUE THREAD

**Issue by itsjoekent** [DoSomethingArchive/legacy-website#6631]
#### Current Behavior
- Staff pick is not properly translated on both Diego's ( @weerd ) & my local machine
- This is causing the front page to freak out with errors
#### Desired Behavior
- The staff pick field should be translatable & the homepage module should stop freaking out
#### Steps to Replicate
- Visit the homepage on local machine 
#### Why This Matters

Not clear why it doesn't work locally, but not a good sign?
#### Relevant Screenshots + Links

Diego DPM()'ing the staff pick field & lang code in the homepage module
![image](https://cloud.githubusercontent.com/assets/897368/16316830/35525ff6-3956-11e6-9081-1e7055e3f347.png)

Same results for me locally 
<img width="1421" alt="screen shot 2016-06-23 at 3 22 14 pm" src="https://cloud.githubusercontent.com/assets/897368/16316838/4ee4913c-3956-11e6-9978-7fb51ae8e6ac.png">

Prod screenshot
<img width="440" alt="screen shot 2016-06-23 at 3 18 36 pm" src="https://cloud.githubusercontent.com/assets/897368/16316845/55ac5e5a-3956-11e6-9346-01c9e0f71f0d.png">

Local screenshot
<img width="325" alt="screen shot 2016-06-23 at 3 23 14 pm" src="https://cloud.githubusercontent.com/assets/897368/16316858/7576dd1e-3956-11e6-8229-67ef3b80fb84.png">

Script output

```
dosomething@dev:/var/www/dev.dosomething.org/html$ drush ds-global-field-fix field_staff_pick
Replacing field values for field_staff_pick.
Data successfully processed.                                                             [status]
```

Relevant PR https://github.com/DoSomething/phoenix/pull/6539

**Comment by angaither** [comment]
@deadlybutter if this is correct on stage, just pull a more recent db?

**Comment by weerd** [comment]
How recently was it corrected on stage? I pulled down DB this past Tuesday and getting this issue...

**Comment by angaither** [comment]
3:15 pm https://dosomething.slack.com/archives/phoenix/p1466709316000529

**Comment by weerd** [comment]
haha

**Comment by weerd** [comment]
<img width="1271" alt="screen shot 2016-06-23 at 12 45 51 pm" src="https://cloud.githubusercontent.com/assets/105849/16317615/c9150aec-3940-11e6-867b-79376f77793e.png">

Doesn't happen 9 times, but still getting this after a complete new run of `ds build` and `ds pull stage --db`

**Comment by angaither** [comment]
Refs #6724

**Comment by angaither** [comment]
fixed by #6792
